src/screens/Grommet.js

```diff
@@ -112,6 +112,16 @@ const GrommetPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="drop">
+          <Description>
+            When using `ContainerTargetContext` drop will be positioned
+            correctly next to target.{' '}
```

Comment:
When enabled additional checks will be performed on the target of the Drop to more accurately determine its position. This is helpful in situations where the target is within a containing block that has a transform, perspective or filter css property applied.

 of the drop will be calculated before determining the Drop position.